sw/host/opentitanlib/src/crypto/spx.rs

```diff
@@ -34,6 +34,14 @@ pub trait SpxPublicKeyPart {
         let full_message = &[domain_sep, message].concat();
         Ok(sphincsplus::spx_verify(self.pk(), &sig.0, full_message)?)
     }
+
+    // Verify a message signature in pre-hashed mode, returning Ok(()) if the
+    // signature matches.
+    fn verify_prehash(&self, oid: &[u8], message: &[u8], sig: &SpxSignature) -> Result<()> {
+        let domain_sep = &[1u8, 0u8];
```

Comment:
I noticed that the `domain_sep` in the verify_prehash is different from the one in the `sign_prehash` function: `[1, 8]` vs `[0, 8]`.  Typo?

Reply:
Yep, a typo. Fixed and I've now added a unit test here that makes sure `sign_prehash` and `verify_prehash` agree!